### PR TITLE
Add default project name and output path on restart wizard

### LINKED ISSUE #1031: Clicking "Create New Project" doesn't auto-populate project name and output path anymore

### PR DESCRIPTION
**Pull request by streetcoder123**
Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/32457

**Changes**
- Now, when you click 'restart wizard' or 'create new Project', the Project name and output path should again be populated for you.

closes #1031

### CODE REVIEW

**Review by Tanya0609 — DISMISSED** [review]
it works!

**Review by SahilTara — DISMISSED** [review]

**Comment by SahilTara** [comment]
Lots of random spacing changes on wizardServant.ts! Good that it's consistent now though.

File: src/client/src/containers/ProjectNameAndOutput/index.tsx [hunk 1]
```diff
@@ -154,8 +168,8 @@ const ProjectNameAndOutput = (props: Props) => {
         )}
         {projectNameMaxLength && (
           <div className={styles.errorMessage}>
-            {props.intl.formatMessage(messages
-              .nameTooLong as FormattedMessage.MessageDescriptor,
+            {props.intl.formatMessage(
+              messages.nameTooLong as FormattedMessage.MessageDescriptor,
```
Comment (jcoc611-microsoft) [inline src/client/src/containers/ProjectNameAndOutput/index.tsx:172]: still don't understand why we get these phantom diffs...do some people not run prettier when they commit?

Comment (streetcoder123) [reply src/client/src/containers/ProjectNameAndOutput/index.tsx:172]: Yeah, maybe some do not have automatic `format on save` enabled. I'll ask the team about this

File: src/extension/src/utils/defaults.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    while (
+      fs.existsSync(path.join(outputPath, newAppName)) &&
+      count <= PROJECT_NAME_VALIDATION_LIMIT
+    ) {
+      newAppName = `${defaultAppName}${count}`;
+      count++;
+    }
+    if (count > PROJECT_NAME_VALIDATION_LIMIT) {
+      return Promise.resolve({ payload: "" });
```
Comment (jcoc611-microsoft) [inline src/extension/src/utils/defaults.ts]: You don't need `Promise.resolve(...)` when the function is async. It gets added automatically.

Comment (streetcoder123) [reply src/extension/src/utils/defaults.ts]: Whoops! Better to have it like this here: `newAppName = ""` and then we can return just once :)
Thanks

**Review by jcoc611-microsoft — APPROVED** [review]

**Review by jsondoo — APPROVED** [review]

**Review by SahilTara — APPROVED** [review]

**Pull request merged**